doozer/doozerlib/constants.py

```diff
@@ -1,4 +1,5 @@
 RC_BASE_URL = "https://{arch}.ocp.releases.ci.openshift.org"
+RC_BASE_PRIV_URL = "https://openshift-release{arch}-priv.apps.ci.l2s4.p1.openshiftapps.com"
```

Comment:
```suggestion
RC_BASE_PRIV_URL = "https://{arch}.ocp.internal.releases.ci.openshift.org/"
```

Reply:
done

Reply:
Thanks for finding the URL, makes things much easier now